sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_bleu/_bleu.py

```diff
@@ -51,9 +52,20 @@ class BleuScoreEvaluator:
         }
     """
 
+    _ID = "bleu"
+
     def __init__(self):
         self._async_evaluator = _AsyncBleuScoreEvaluator()
 
+    @experimental
+    @classmethod
```

Comment:
https://stackoverflow.com/questions/128573/using-property-on-classmethods/64738850#64738850

according to this, we can't do this in python 3.13 anymore (and it was deprecated in python 3.11)

Reply:
Updated, thanks for sharing this information